project/yang2829/README.rst

```diff
@@ -0,0 +1,63 @@
+Voronoi Diagram
+===============
+
+Description
+------------
+GitHub: https://github.com/yang2829/Voronoi-Diagram
+
+Problem to Solve
+----------------
+The Voronoi diagram is a decomposition of a region that any point in a sub-region is closest to the site of the sub-region.
+
+.. image:: https://upload.wikimedia.org/wikipedia/commons/5/54/Euclidean_Voronoi_diagram.svg
+   :width: 40%
+
+Voronoi diagrams are widely used in fields such as Geographic Information Systems (GIS) and Computer Graphics.
+This project is limited to 2-dimensional space. It may use Fortune's algorithm to solve it.
```

Comment:
It is good to have an explicit constraint.  It will help you design and implement.